modules/disk-uploader/pkg/secrets/secrets.go

```diff
@@ -22,7 +24,7 @@ func CreateVirtualMachineExportSecret(k8sClient kubernetes.Interface, namespace,
 
 	v1Secret := &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
+			Name:      generateRandomName(name),
```

Comment:
I aimed to have similar naming to `generateSecureRandomString` above. I don't mind naming it `generateName`, but why it should be public function?

Reply:
I renamed it to `generateName` in this [revision](https://github.com/kubevirt/kubevirt-tekton-tasks/compare/b78b6e03d0d98671a3773378e389824d1f732f98..9b539e15bb30d47802af7f29e4ac29396b5e8fda).

Reply:
I meant, why are we not using the `GenerateName` feature offered by K8S? The field is named `GenerateName` instead of name and will append a random suffix.

Reply:
Do you mean to `GenerateName` that is part of ObjectMeta? For example:
```
v1Secret := &corev1.Secret{
		ObjectMeta: metav1.ObjectMeta{
			GenerateName: name + "-",
...
}
```